projects/angular-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cinchy-co/angular-sdk",
-  "version": "5.1.4",
+  "version": "6.0.0-alpha.0",
```

Comment:
We typically use the `.rc-` prefix for this project. There are scripts in package.json that facilitate this workflow if that's what you're looking for.